### PR TITLE
Feature: Gem Views Report 

### PR DESCRIPTION
**Pull request by rostyslav-horytskyi**
image

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
-  namespace NodeJS {
+  namespace TagJS {
```
Comment (slavik-chapelskyi) [inline src/types.ts:5]: What is TagJS?

File: pages/index.scss [hunk 2]
```diff
@@ -13,5 +13,21 @@ body {
   line-height: 1.5;
   font-family: 'Open Sans', serif;
   height: 100vh;
-  overflow: hidden;
+  // overflow: hidden;
```
Comment (pihorb) [inline pages/index.scss:16]: comment

File: src/components/BarChart/BarChart.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import React, {FC} from 'react';
+import Chart from 'react-google-charts';
+import {Props} from '../../types';
```
Comment (pihorb) [inline src/components/BarChart/BarChart.tsx:3]: Please use more descriptive name

File: pages/index.scss (hunk 2 above)
Comment (pihorb) [reply pages/index.scss:16]: Please, remove comment

Comment (rostyslav-horytskyi) [reply pages/index.scss]: Sorry, I forgot

**Review by pihorb — APPROVED** [review]

**Pull request merged**